<launch>

  <arg name="map_frame" default="map" />
  <arg name="base_frame" default="base_link" />
  <arg name="publish_rate" default="10" />

  <node pkg="trajectory_visualizer" type="trajectory_saver_node" name="trajectory_saver"
    output="screen">
    <param name="map_frame" value="$(arg map_frame)" />
    <param name="base_frame" value="$(arg base_frame)" />
    <param name="publish_rate" value="$(arg publish_rate)" />
  </node>
</launch>
